src/js/modules/graph.js

```diff
@@ -131,14 +131,15 @@ var captionFor = function (d) {
 
 export function forcedGraph (svgElement, graph) {
   var shapeSize = 2000
-  var strength = -1000
+  var strength = -200
+  var linkStrength = 0.2
+  var linkDistance = 90
```

Comment:
@adam-cowley I've tweaked the numbers a bit to make sure that nodes won't push each other too much